### ISSUE TITLE
Problem with initial equation in Building interface when standAlone=false

### ISSUE THREAD

**Issue by cprotopa** [open-ideas/IDEAS#671]
This is a problem due to an old commit: [1909c57](https://github.com/open-ideas/IDEAS/commit/1909c5760a9d3ab60c385a8605083666c7a09838#diff-3ac73bb36aaec782150393bce95a69b4). Unfortunately we only now noticed. 
In `IDEAS.Templates.Interfaces.Building` the following initial equation was introduced to get rid of warnings (if not mistaken): `voltageSource.pin_p.reference.gamma=0;`
When `standAlone=false`, this gives an error because `voltageSource` is conditional on `standAlone`. 
Removing the equation helps. Didn't seem to work with `if standAlone then` structure. Could perhaps the warning be fixed otherwise?

**Comment by Mathadon** [comment]
That seems correct. The cleanest way to deal with this would probably be to remove all grid equations altogether when `standAlone = false`, such that no initial equation is needed. 

Do you see any problems with that idea?

**Comment by cprotopa** [comment]
In this particular model there are only few grid **equations**, which are already conditional on `standAlone`: 
```
if standAlone then
    connect(voltageSource.pin_p, ground.pin);
    connect(inHomeGrid.pinSingle, voltageSource.pin_n) ;
  else
    connect(inHomeGrid.pinSingle, plugFeeder) ;
  end if;
```
These didn't cause any problem. Somehow the same approach,  using the `if` statement, didn't work for the  **initial equation**. Maybe because it's the only initial equation, and then there is none? Can it be implemented another way, say as a start value? 
So, NO, no problem having the equations removed for `standAlone=false`, if we manage.

**Comment by Mathadon** [comment]
@cprotopa can you verify if the proposed solution:
https://github.com/open-ideas/IDEAS/commit/e14097c7449df9b78660bc130fe34d0a1cbfcfa0
fixes your problem?

thanks!

**Comment by cprotopa** [comment]
Yes, works. At least for the case I had in mind. Thanks.

**Comment by Mathadon** [comment]
closed in #687
